src/collective/exportimport/import_content.py

```diff
@@ -399,6 +407,120 @@ def import_new_content(self, data):  # noqa: C901
+            try:
+                new = deserializer(validate_all=False, data=version)
+            except Exception as error:
+                logger.warning(
+                    "cannot deserialize {}: {}".format(item["@id"], repr(error))
+                )
+                return
+
+            self.save_revision(new, version, initial)
+
+        # Finally create the current version
+        new = container.get(item["id"])
+        deserializer = getMultiAdapter((new, self.request), IDeserializeFromJson)
+        try:
+            new = deserializer(validate_all=False, data=item)
+        except Exception as error:
+            logger.warning("cannot deserialize {}: {}".format(item["@id"], repr(error)))
+            return
+
+        self.import_blob_paths(new, item)
+        self.import_constrains(new, item)
+        self.global_obj_hook(new, item)
+        self.custom_obj_hook(new, item)
+
+        if item["review_state"] and item["review_state"] != "private":
+            if portal_workflow.getChainFor(new):
+                try:
+                    api.content.transition(to_state=item["review_state"], obj=new)
+                except InvalidParameterError as e:
+                    logger.info(e)
+
+        # Import workflow_history last to drop entries created during import
+        self.import_workflow_history(new, item)
+
+        # Set modification and creation-date as a custom attribute as last step.
+        # These are reused and dropped in ResetModifiedAndCreatedDate
+        modified = item.get("modified", item.get("modification_date", None))
+        if modified:
+            modification_date = DateTime(dateutil.parser.parse(modified))
+            new.modification_date = modification_date
+            new.aq_base.modification_date_migrated = modification_date
+        created = item.get("created", item.get("creation_date", None))
+        if created:
+            creation_date = DateTime(dateutil.parser.parse(created))
+            new.creation_date = creation_date
+            new.aq_base.creation_date_migrated = creation_date
+
+        self.save_revision(new, item)
+        logger.info("Created item: {} {} with {} old versions".format(item["@type"], new.absolute_url(), len(item["exportimport.versions"])))
+
+        # TODO: Maybe reset versioning?
+        # if is_versioned:
+        #     behaviors = list(fti.behaviors)
+        #     behaviors.append('plone.versioning')
+        #     fti.behaviors = behaviors
+        return new
+
+    def save_revision(self, obj, item, initial=False):
+        """Save revision manually to set dates and changenote from exported data."""
+        rt = api.portal.get_tool("portal_repository")
+
+        modified = dateutil.parser.parse(item["modified"])
+        # add one millisecond to prevent created being before first revision
+        modified  = modified + timedelta(milliseconds=1)
+        timestamp = datetime.timestamp(modified)
```

Comment:
`datetime.timestamp` is only available from python 3.3 up. For python 2 we could use `time.mktime.`

```suggestion
         try:
            timestamp = datetime.timestamp(modified)
        except AttributeError:
            # support python 2.7 
            import time
            timestamp = time.mktime(modified.timetuple())
```

Reply:
fixed